.github/workflows/auto-publish.yml

```
@@ -1,39 +1,72 @@
-      - name: Run Build Scripts
+      # 步骤4: 获取pnpm缓存目录路径
+      - name: Get pnpm store directory
+        id: pnpm-cache
         run: |
-          npm i --legacy-peer-deps
-          npm run build
-          npm run release
+          echo "pnpm_cache_dir=$(pnpm store path)" >> $GITHUB_OUTPUT
+
+      # 步骤5: 配置pnpm缓存
+      - uses: actions/cache@v3
```

actionlint (1.7.7)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)